### PR TITLE
Create IoT_code.ino

### PR DESCRIPTION
**Pull request by Awesome12-arch**

### CODE REVIEW

File: soilSensor/base-config/IoT_code.ino [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+#include <String.h>
+
+//Ubidots account data
+String token = "  ";      //your token to post value
+String temp_id="  ";                                 //ID of your temperature variable
+String hum_id="  ";                                 //ID of your humidity variable
+String soil_id="  ";                                 //ID of your soil variable
+String light_id="  ";                             //ID of your light variable
+
+//sensor output connections
```
Comment (Awesome12-arch) [inline soilSensor/base-config/IoT_code.ino:12]: Comments for the IoT code

Comment (Awesome12-arch) [reply soilSensor/base-config/IoT_code.ino:12]: Yes

**Comment by Gaurav-Ramakrishna** [comment]
@Awesome12-arch thank you for your contribution and we would love to know more about your it. Can you please create a new [feature request](https://github.com/Call-for-Code/Liquid-Prep/issues/new/choose) and give us some details about your contribution like objective, pre-requisites, setup instructions, and how to execute your contribution.
Thanks

**Comment by krook** [comment]
@Awesome12-arch Thanks again for your contribution. Can you please reply to the questions above?

**Pull request closed without merging**